index.html

```diff
@@ -1,229 +1,300 @@
+            
+            <li class="icons">
+              <img class="ang-logo" src="./images/angellist.svg" alt="An icon logo of angellist">
+            </li>
+            
+            <li class="icons"><a href="#"><i class="bi bi-github"></i></a></li>
+            
+            <li class="icons">
+              <img class="ang-logo" src="./images/medium.svg" alt="An icon logo of medium">
+            </li>
+          </ul>
+
+          <button class="resume"><a href="#">Get my resume</a></button>
+        </section>
+
+          <div class="height-1">
+            <ul>
+              <li >
+                <div class="lang-div frame">
+                  <p class="lang">Languages</p>
+                  <button class="lang-btn"><i class="fa-solid fa-2x fa-angle-down"></i></button>
+                </div>
+                <ul>
+                  <li class="ellipse">
+                    <img src="./images/Ellipse 1.png" alt="JavaScript logo">
+                    <p>JavaScript</p>
+  
+                  </li>
+  
+                  <li class="ellipse">
+                    <img src="./images/Ellipse 1 (1).png" alt="HTML logo">
+                    <p>HTML</p>
+                  </li>
+  
+                  <li class="ellipse">
+                    <img src="./images/Ellipse 1 (2).png" alt="CSS logo">
+                    <p>CSS</p>
+                  </li>
+                </ul>
               </li>
               <li>
-                <img src="assets/image/htmlLogo.png" alt="Html" />
-                <a href="#">HTML</a>
+                <div class="frame">
+                  <p class="frame-title" >Frameworks</p>
+                  <button class="lang-btn"><i class="fa-solid fa-2x fa-angle-right"></i></button>
+                </div>
+                <hr>
               </li>
               <li>
-                <img src="assets/image/css_logo.png" alt="CSS" />
-                <a href="#">CSS</a>
+                <div class="frame">
+                  <p class="skill-title">Skills</p>
+                  <button class="lang-btn"><i class="fa-solid fa-2x fa-angle-right"></i></button>
+                </div>
               </li>
+              <hr>
             </ul>
-          </li>
-          <li class="lang-exp">
-            <div class="no-1">
-              <p>Frameworks</p>
-              <img src="assets/image/arrow_bottom.png" alt="arrow_bottom" />
-            </div>
-            <hr />
-          </li>
-
-          <li class="lang-exp">
-            <div class="no-1">
-              <p>Skills</p>
-              <img src="assets/image/arrow_bottom.png" alt="arrow_bottom1" />
-            </div>
-            <hr />
-          </li>
-        </ul>
-      </div>
-    </div>
-  </section>
-</body>
-
+              <!-------------------------------------- Contact form------------------------------->
+            <div class="about-contact">
+              <div class="title-container">
```

Comment:
[OPTIONAL] This is optional but recommended so kindly use the `section` tag instead of the `div` tag because the `section` element can be used to divide up a web page into standalone sections of related content. Using the `section` element over a generic container element like `div` can help make your code more `accessible` and `understandable` to `search engines`, `browsers`, `assistive technologies`, and `other developers`.

Reply:
Resolve